docs/training-apis/api-pytorch-advanced.txt

```diff
@@ -0,0 +1,147 @@
+########################
+ PyTorch API - Advanced
+########################
+
+*******************
+ Gradient Clipping
+*******************
+
+Users need to pass a gradient clipping function to
+:meth:`~determined.pytorch.PyTorchTrialContext.step_optimizer`.
+
+.. _pytorch-custom-reducers:
+
+******************
+ Reducing Metrics
+******************
+
+Determined supports proper reduction of arbitrary training and validation metrics, even during
+distributed training, by allowing users to define custom reducers. Custom reducers can be either a
+function or an implementation of the :class:`determined.pytorch.MetricReducer` interface. See
+:meth:`determined.pytorch.PyTorchTrialContext.wrap_reducer` for more details.
+
+.. _reproducible-dataset:
+
+***********************
+ Reproducible Datasets
```

Comment:
Before, the api reference for pytorch.samplers and this explanation of how and why you would use them were always together.  Them being separated and not linked in any way seems more confusing to me.

Also, now that they are separated, this title makes it sound like having Reproducible Data is an advanced thing, rather than something you only need to concern yourself with when you have a totally custom dataset.